### ISSUE TITLE
Seeking YouTube video removes LineageOS AudioFX equalizer settings

### ISSUE THREAD

**Issue by 0x416c6578** [TeamNewPipe/NewPipe#7033]
### Checklist

- [x] I am using the latest version - 0.21.9 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug
1. Play video with an equalizer profile enabled in the AudioFX app
2. Double tap to seek forward / back (this works for any time skip, also with fast seek enabled), or just tap on the seekbar to skip to a different part of the video
3. Equalizer will stop working, audio returns to un-equalized volume
- This bug seems to occur when loading from an un-preloaded part of a video (i.e. the seekbar doesn't have the lighter red colour), or when the video buffers (i.e. the buffering wheel appears on screen). When the skip is instant (i.e. no buffering happens), the bug doesn't occur. This isn't 100% effective in causing the bug however, so there might be some race condition happening. It seems to be when there is a shorter buffering time, the crash happens (and the EQ settings are removed), and when there is a longer buffering time, the EQ settings don't get removed
- When closing the video player and loading the video again, the EQ settings are restored
- The bug doesn't happen when seeking whilst in background mode
- It also seems to be YouTube specific

### Actual behavior
- Audio EQ balance profile stops working after video seek

### Expected behavior
- Audio EQ balance won't change on video seek

### Logs

Nothing in LogCat indicates that AudioFX / AudioServer is crashing, however I suspect this is probably device / LineageOS version specific.

### Device info

 - Android version/Custom ROM version: LineageOS 18.1
 - Device model: OnePlus 5T

**Comment by mhmdanas** [comment]
Can you try the APK in #6993 and tell us if the issue is present there?

**Comment by 0x416c6578** [comment]
Hmm, ive tried both the APK and building a release APK and the bug occurs in both still.

**Comment by 0x416c6578** [comment]
Calling notifyAudioSessionUpdate(active:true) on forward/back seek does seem to fix if the EQ does fail, however probably 1/5 times the EQ turns off like before until I skip again, at which point it is reenabled.

**Comment by 0x416c6578** [comment]
Yeah it could be a LineageOS problem, however I feel like there could be a workaround with a call to `notifyAudioSessionUpdate`, since my experiments somewhat fixed the bug. I have noticed very similar behaviour regarding EQ settings being disabled in [musicolet](https://play.google.com/store/apps/details?id=in.krosbits.musicolet&hl=en_US&gl=US), and have submitted a bug report there. If you do think it is a LineageOS problem, I can close this issue if you want.

**Comment by comicsansgreenki** [comment]
I'm also running LineageOS on a Picel 2, and I'm noticing that the player just bypasses AudioFX entirely.

**Comment by ShareASmile** [comment]
Is it still an issue on 0.26.1?
& after a year or si with LineageOS updates, did anything change ?
